mantle/cmd/plume/release.go

```diff
@@ -169,12 +169,12 @@ func makeReleaseAMIsPublic(rel release.Release) bool {
 				Region:          region,
 			})
 			if err != nil {
-				plog.Warningf("creating AWS API for modifying launch permissions: %v", err)
+				plog.Warningf("creating AWS API for region %s modifying launch permissions: %v", region, err)
 				at_least_one_failed = true
 				continue
 			}
 
-			plog.Noticef("making AMI %s public", ami.Image)
+			plog.Noticef("making AMI %s in region %s public", ami.Image, region)
```

Comment:
Nice. I was actually thinking the other day as I was watching one of these run that it would be nice to know which region it was operating on when seeing these messages!